docs/sections/learn/caching.md

```diff
@@ -42,8 +42,54 @@ Finally, if we decide to run the same `Pipeline` after it has finished completel
 
 ### Serialization
 
-!!! NOTE
-    WORK IN PROGRESS
+Let's see what get's serialized by looking at a sample `Pipeline`'s cached folder:
+
+```bash
+$ tree ~/.cache/distilabel/pipelines/73ca3f6b7a613fb9694db7631cc038d379f1f533
+├── batch_manager.json
```

Comment:
I was checking because this wasn't the initial idea. Currently, once we call `Pipeline.run`, it will create the signature of the pipeline, and in case it already exists (was cached at some point), it loads it. It could be useful to have it like you mention, but we need to update the code, I think it could be better done in the next release. Could you open an issue for this feature?